bin/istio-proxy-cfg

```diff
@@ -69,7 +69,7 @@ function check_endof_cmdline {
 
 function get_pilot_url {
     if [[ -z $PILOT_URL ]]; then
-        pilot_ipport=(`kubectl get svc -n ${ISTIO_NAMESPACE} | grep ${ISTIO_PILOT} | awk '{print $2,$4}'`)
+        pilot_ipport=(`kubectl get svc -n ${ISTIO_NAMESPACE} | grep ${ISTIO_PILOT} | awk '{print $3,$5}'`)
```

Comment:
This is great. I'd like to see a follow-up change to make it agnostic to k8s version.